modules/help_editor/jsx/help_editor_helper.js

```diff
@@ -0,0 +1,84 @@
+/* eslint new-cap: ["error", {capIsNewExceptions: ["RMarkdown"]}]*/
```

Comment:
@laemtl shouldn't have this line

Reply:
I agree. I was just not sure if renaming RMarkdown was desired as it's a convention accross a lot of modules.